lstchain/image/cleaning/cleaning.py

```diff
@@ -0,0 +1,136 @@
+import tables
+import numpy as np
+
+from lstchain.io.io import dl1_params_tel_mon_ped_key, dl1_params_tel_mon_cal_key
+
+
+def tailcuts_clean_with_pedestal_threshold(
+    geom,
+    image,
+    interleaved_pedestal_thresh,
+    picture_thresh=7,
```

Comment:
maybe include here the standard
`picture_thresh=6`
`boundary_thresh=3`
?

Reply:
> maybe include here the standard
> `picture_thresh=6`
> `boundary_thresh=3`
> ?

Actually they are a bit too low for real data, after the "re-calibration" (correction for pulse tails) that we introduced in the latest lstchain release. So I would put there whatever Julian and Pawel think is reasonable according to their tests.

Reply:
I will add script to reprocess data, where we will be able give threshold values in config file.